app/scripts/controllers/landingPage.js

```diff
@@ -47,4 +61,31 @@ angular.module('openshiftConsole')
       // add the item to recently-viewed. No-op if the dialog is not open.
       addTemplateToRecentlyViewed();
     });
+
+    function dataLoaded() {
+      if (tourConfig && tourConfig.enabled && tourConfig.steps) {
```

Comment:
Would be good to calculate this once above and remember so we don't have the same logic twice in the same controller.

I think an early return is easier to understand here instead of nested `if` statements.

```js
if (!tourEnabled) {
  return;
}

if ($routeParams.startTour) ...
```